src/jaxsim/parsers/rod/utils.py

```diff
@@ -206,3 +218,54 @@ def fibonacci_sphere(samples: int) -> npt.NDArray:
     return descriptions.SphereCollision(
         collidable_points=collidable_points, center=center_wrt_link
     )
+
+
+def create_mesh_collision(
+    collision: rod.Collision,
+    link_description: descriptions.LinkDescription,
+    method: MeshMappingMethods = MeshMappingMethods.VertexExtraction,
+    nsamples: int = 1000,
+) -> descriptions.MeshCollision:
+    file = pathlib.Path(resolve_local_uri(uri=collision.geometry.mesh.uri))
+    _file_type = file.suffix.replace(".", "")
+    mesh = trimesh.load_mesh(file, file_type=_file_type)
+
+    if mesh.is_empty:
+        logging.warning(f"Mesh {collision.geometry.mesh.uri} is empty, ignoring it")
+        return
+
+    mesh.apply_scale(collision.geometry.mesh.scale)
+    f"===> Loading mesh {collision.geometry.mesh.uri} with scale {collision.geometry.mesh.scale}, file type '{_file_type}'"
+
+    # Extract the points from the mesh to use as colliders according to the provided method
+    match method:
+        case MeshMappingMethods.VertexExtraction:
+            points = mesh.vertices
+        case MeshMappingMethods.RandomSurfaceSampling:
+            points = mesh.sample(nsamples)
+        case MeshMappingMethods.UniformSurfaceSampling:
+            raise NotImplementedError("Uniform surface sampling not implemented")
+        case _:
+            raise ValueError("Invalid mesh mapping method")
+
+    points = mesh.vertices
+    H = (
+        collision.pose.transform() if collision.pose is not None else np.eye(4)
+    )  # pose of the collision object
```

Comment:
nit: can you move comments in a new line before the variable assignment instead of in the end of its line?